runtime/doc/if_lua.txt

```diff
@@ -552,7 +568,17 @@ tsnode:named_descendant_for_range(start_row, start_col, end_row, end_col)
 	range of (row, column) positions
 
 ------------------------------------------------------------------------------
-VIM							*lua-util*
+VIM							*lua-builtin*
+
+vim.api.{func}({...})					*vim.api*
+        Invokes Nvim |API| function {func} with arguments {...}.
+        Example: call the "nvim_get_current_line()" API function: >
+            print(tostring(vim.api.nvim_get_current_line()))
+
+vim.call({func}, {...})					*vim.call()*
+        Invokes |vim-function| or |user-function| {func} with arguments {...}.
+        See also |vim.fn|. Equivalent to: >
+            vim.fn[func]({...})
```

Comment:
we should try to keep this in alphabetical order. I didn't do that fully here, but just moved `vim.call` near the top.